### PR TITLE
Simplify imports with post qualified style

### PR DESCRIPTION
**Pull request by JordanMartinez**
**Description of the change**

First part of #4450. I'm breaking this up over multiple PRs because of how many files this will touch.

Changes made across `app`, `src`, and `tests`:
```diff
-import qualified Foo
+import Foo qualified
-import qualified Foo as F
+import Foo qualified as F
-import qualified Foo    as F
+import Foo qualified as F
-import           Foo
+import Foo
-import           Foo         (bar)
+import Foo (bar)
-import           "monad-logger" Foo
+import "monad-logger" Foo
```

---

**Checklist:**

- [x] Added a file to CHANGELOG.d for this PR (see CHANGELOG.d/README.md)
- [ ] Added myself to CONTRIBUTORS.md (if this is my first contribution)
- [ ] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by rhendric** [comment]
Let's remove `-Wno-prepositive-qualified-module` from the cabal file too.

File: purescript.cabal [hunk 1]
```diff
@@ -118,6 +118,7 @@ common defaults
     TupleSections
     TypeFamilies
     ViewPatterns
+    ImportQualifiedPost
```
Comment (rhendric) [inline purescript.cabal:121]: Alphabetical position?

**Comment by JordanMartinez** [comment]
Ah... I forgot about the `app` dir...

**Comment by JordanMartinez** [comment]
Ok. CI builds now.

**Review by rhendric — APPROVED** [review]
Have verified this PR makes no changes to source files other than whitespace and moving the position of `qualified`.

**Pull request merged**